.github/sync-repo-settings.yaml

```diff
@@ -9,16 +9,17 @@ branchProtectionRules:
   requiredStatusCheckContexts:
     - 'cla/google'
     - 'OwlBot Post Processor'
-    # TODO(tswast): enable once we have tests migrated from google-cloud-bigquery
-    # - 'lint'
+    - 'lint'
+    # TODO: Enable once the docs tests ready.
     # - 'docs'
-    # - 'unit (3.8)'
-    # - 'unit (3.9)'
-    # - 'unit (3.10)'
-    # - 'unit (3.11)'
-    # - 'unit (3.12)'
-    # - 'cover'
-    # - 'Kokoro'
+    - 'unit (3.8)'
+    - 'unit (3.9)'
+    - 'unit (3.10)'
+    - 'unit (3.11)'
+    - 'unit (3.12)'
+    - 'cover'
+    - 'Kokoro'
```

Comment:
This one is a duplicate with all our system tests. Let's add the specific system tests here instead.

Reply:
Addressed. Thanks!